### DISCUSSION TITLE
Effective Resistance

### DISCUSSION THREAD

**Discussion by hamidehhhs** [networkit/networkit#992] [Q&A]
I wonder if it is  possible to compute  or approximate effective resistance between two pairs of nodes in networkit ?

**Comment by hmeyerhenke** [comment]
If you go to the Documentation on the homepage, then search for "commute". Commute time and resistance distance are the same except for the volume(G) factor.

**Comment by hamidehhhs** [comment]
Thanks for your reply.
I have tested that function on a network with around 400*10^6 edges but it failed because of too much memory consumption. 
I wished there is any alternative!

**Comment by hmeyerhenke** [comment]
There is. Use CG to set up and solve the corresponding linear system (see numerics folder). For a few pairs this should work within a few minutes, I would assume. LAMG as solver is probably too space-consuming on such a graph, depending on the RAM you have...
